cmd/otelcontribcol/go.mod

```diff
@@ -2,13 +2,17 @@
 
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontribcol
 
-go 1.22.0
-
-toolchain go1.22.7
+go 1.22
```

Comment:
I believe this is causing the failures of to `go mod tidy`. However, since this file is generated by core's `cmd/builder`, I'm not entirely sure what to do here.